Alisha/data/lab4.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>Change the title to Visualization of 1D data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>